apps/analysis/tests/test_apis.py

```diff
@@ -116,6 +122,82 @@ def test_create_analytical_statement(self):
         r_data = response.json()
         self.assertEqual(r_data['statement'], data['statement'])
 
+    def test_create_analytical_statement_greater_than_30(self):
+        user = self.create_user()
+        project = self.create_project()
+        project.add_member(user)
+        entry = self.create(Entry)
+        analysis = self.create(Analysis, project=project)
+        pillar = self.create(AnalysisPillar, analysis=analysis)
+        analytical_statement1 = self.create(AnalyticalStatement, analysis_pillar=pillar)
+        with self.assertRaises(ValidationError):
+            for _ in range(ANALYTICAL_STATEMENT_COUNT + 1):
+                self.create(AnalyticalStatement, analysis_pillar=pillar)
+        new_count = AnalyticalStatement.objects.filter(analysis_pillar=pillar).count()
+        self.assertEqual(new_count, ANALYTICAL_STATEMENT_COUNT)
```

Comment:
Lets remove this.

And add a test where client tries calling the api with more than `ANALYTICAL_STATEMENT_COUNT` statements for a given pillar.

Reply:
Done